lib/screens/register_user/widgets/registration_form.dart

```diff
@@ -9,34 +9,66 @@ class RegisterUserForm extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Form(
-      key: _formKey,
-      child: Column(
-        children: [
-          TextFormField(
-            controller: mumbleUserController,
-            validator: (value) {
-          final valRegex = RegExp(r'^[a-zA-Z0-9_]{3,}$');
-          final whitespaceRegex = RegExp(r"\W");
-          return (value == null ||
-                  value.isEmpty ||
-                  !valRegex.hasMatch(value) ||
-                  value.startsWith(whitespaceRegex) ||
-                  value.endsWith("_"))
-              ? "Please enter a valid user name."
-              : null;
-            },
+    return Padding(
+        padding: EdgeInsets.symmetric(
+          horizontal:
+          MediaQuery.of(context).size.width * 0.05,
+        ),
+        child: Form(
+          key: _formKey,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              TextFormField(
```

Comment:
Also please add a Text widget informing the user that `Please only use a combination of lowercase letters, uppercase letters and underscored (_)`.

Reply:
This is ready, i fixed it.